includes/db.inc

```diff
@@ -35,9 +35,14 @@ function islandora_usage_stats_pid_id($pid) {
  */
 function islandora_usage_stats_pid_to_db($pid) {
```

Comment:
@bryjbrown Also, I'm not sure you're reading the ternary correctly. It's a form of parameter overloading, to allow multiple types to be passed in, either the string PID, or a loaded object.

These two should be equivalent:
```php
$object = $pid instanceof AbstractObject ? $pid : islandora_object_load($pid);
```
and:
```
if ($pid instanceof AbstractObject) {
  $object = $pid;
}
else {
  $object = islandora_object_load($pid);
}
```